prac_05/word_occurrences.py

```diff
@@ -0,0 +1,27 @@
+# Adrianto Hartono
+# 14347930
+
+"""
+Word Occurrences
+Estimate: 30 minutes
+Actual: 40 minutes
+"""
+
+user_input= str(input("Text: "))
+text = user_input.split()
+length = 0
```

Comment:
Had trouble with this one hope you don't mind me referencing it 😊

Reply:
thank you, feel free to reference 😁

Reply:
Nicely done! It works!

Reply:
thanks for the feedback 🙏